fraim/core/workflows/chunk_processing.py

```diff
@@ -106,6 +121,10 @@ async def process_chunk_with_semaphore(chunk: CodeChunk) -> List[T]:
         active_tasks: Set[asyncio.Task] = set()
 
         for chunk in project:
+            self.config.logger.debug("Processing chunk '''\n%s\n'''", chunk)
```

Comment:
This is a lie :p 

I'm not a huge fan of this flag, but while we are iterating and debugging this it's cool. Not something to document, at least.